extensions/sql-migration/scripts/SHIR-auto-configuration.ps1

```diff
@@ -100,11 +100,11 @@ Function Install-IR {
 		Write-OutputAndLog "Integration Runtime found: Version $installedIRVersion is installed on the machine."
 
 		# Perform validation checks for installed IR
-		if ($installedIRVersion -ne $Global:LatestIRVersion) {
-			Write-ErrorAndLog "Installed Integration Runtime's version $($installedIRVersion) does not meet the version $($Global:LatestIRVersion) or above required for configuring."
-			return
-		}
-		Write-OutputAndLog "The installed Integration Runtime satisfies all requirements for successful configuration."
+		# if ($installedIRVersion -ne $Global:LatestIRVersion) {
+		# 	Write-ErrorAndLog "Installed Integration Runtime's version $($installedIRVersion) does not meet the version $($Global:LatestIRVersion) or above required for configuring."
+		# 	return
+		# }
```

Comment:
is this line not needed, if yes let's remove it?

Reply:
This will be used in the next phase